modules/auxiliary/server/mysql_rogue.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env python3
+
+# mysql rogue server requirements
+import socket
+import asyncore
+import asynchat
+import struct
+import logging
+import logging.handlers
+import codecs
+from argparse import ArgumentParser
+from enum import Enum
+import metasploit.module as module
+
+metadata = {
+    'name': 'mysqlrogue',
+    'description': '''
+        The script starts a MySQL server that requests and retrieves files from clients that connect to it.
+    ''',
+    'authors': [
+        'Moritz Nentwig'
+    ],
+    'date': '2021-09-03',
+    'license': 'MSF_LICENSE',
+    'references': [
+        {'type': 'url', 'ref': 'https://github.com/ShielderSec/CVE-2020-11579'},
```

Comment:
```suggestion
        {'type': 'url', 'ref': 'https://github.com/ShielderSec/CVE-2020-11579'},
        {'type': 'url', 'ref': 'https://github.com/Gifts/Rogue-MySql-Server'},
```

A few other useful references:

* https://sansec.io/labs/2019/01/17/adminer-4.6.2-file-disclosure-vulnerability/
* https://sansec.io/labs/2019/01/20/sites-hacked-via-mysql-protocal-flaw/
* https://dev.mysql.com/doc/refman/8.0/en/load-data-local.html